buildbot/buildbot/worker.py

```diff
@@ -217,6 +219,8 @@ def stage_make(db, config):
     with work(db, state.MAKE, state.MAKE_PROGRESS, state.HLS_FINISH) as task:
         _task_config(task, config)
 
+        task['hw_basename'] = task['config'].get('kernel')
```

Comment:
Rather than "overloading" this field's meaning, which was originally meant to control the Makefile-free code generation step, let's instead just display `task['config']['kernel']` in the UI directly. Here's the relevant line of code in the UI template:
https://github.com/cucapra/seashell/blob/79b297306035a24d3af9913cd34f8ad583a529bb/buildbot/buildbot/templates/joblist.html#L45

Reply:
I'm not quite sure whether I resolved this in the cleanest way.